pages/download.html

```diff
@@ -309,6 +309,19 @@ <h3>{% trans "Linux / Source Code" %}</h3>
   </div>
 </div>
 
+<div id="snapshots" class="docs-section container">
+  <h2><a class="headline-link" href="#snapshots">{% blocktrans %}Development Snapshots{% endblocktrans %}</a></h2>
+  <p>
+    {% blocktrans with source_link="https://github.com/mixxxdj/mixxx" windows_snapshot_link="https://downloads.mixxx.org/builds/appveyor/cmake_build/" macos_snapshot_link="https://downloads.mixxx.org/builds/travis/main/" ubuntu_snapshot_link="https://downloads.mixxx.org/builds/travis/main/" %}Want to test and experiment with the latest changes?
```

Comment:
```suggestion
    {% blocktrans with source_link="https://github.com/mixxxdj/mixxx" windows_snapshot_link="https://downloads.mixxx.org/builds/appveyor/cmake_build/?C=M;O=A" macos_snapshot_link="https://downloads.mixxx.org/builds/travis/main/?C=M;O=A" ubuntu_snapshot_link="https://downloads.mixxx.org/builds/travis/main/?C=M;O=A" %}Want to test and experiment with the latest changes?
```
Let's link to the directory listings sorted by date modified. Also, we should change the AppVeyor deployments to give each branch its own folder...

Reply:
I agree, but we can just make a new PR to update the links when we took care of it.